data/pathfinder/paizo/roleplaying_game/ultimate_equipment/ultimate_equipment.pcc

```diff
@@ -51,8 +51,9 @@ EQUIPMENT:ue_equip_general.lst
 EQUIPMENT:ue_equip_magic_items.lst
 EQUIPMOD:ue_equipmods.lst
 KIT:ue_kits.lst
-# ue_spells.lst only required if UC is not loaded - it avoids a dependency there
-SPELL:ue_spells.lst|!PRECAMPAIGN:1,INCLUDES=Ultimate Combat
+# ue_spells_*.lst only required if UC/UM are not loaded - avoids dependencies
+SPELL:ue_spells_uc.lst|!PRECAMPAIGN:1,INCLUDES=Ultimate Combat
```

Comment:
I have nothing against the dependencies removal, however, I am not very keen on duplicating objects. If errata comes out, I need to track down a spell in multiple sources. Is the INCLUDE/EXCLUDE code not up to this task?